enterprise/kernel/src/test/java/org/neo4j/util/TestHelpers.java

```diff
@@ -76,7 +80,15 @@ public static int runBackupToolFromOtherJvmToGetExitCode( File neo4jHome, String
         return new ProcessStreamHandler( process, false ).waitForResult();
     }
 
-    public static String backupAddress( GraphDatabaseAPI graphDatabase )
+    public static String backupAddressCc( GraphDatabaseAPI graphDatabase )
+    {
+        ListenSocketAddress hostnamePort = graphDatabase.getDependencyResolver().resolveDependency( Config.class ).get(
+                listenAddress( "causal_clustering.transaction_listen_address", 6000 ) );
```

Comment:
I think you should be fine with just config.get( CausalClusteringSettings.xxx ).

Reply:
Actually this would cause a circular dependency; It's surprising that the string works though.